src/main/java/lesson12Names/NameChanger.java

```diff
@@ -0,0 +1,28 @@
+package lesson12Names;
+
+import java.util.*;
+
+public class NameChanger {
+
+    public static void main(String[] args) {
+
+        ArrayList<String> namesList = new ArrayList<>(Arrays.asList
+                ("Anna", "Katarzyna", "Tymon", "Tadeusz", "Anna", "Karolina", "Dorota", "Piotr"));
+
+        Set<String> namesSet = new HashSet<>(namesList);
+        System.out.println("Lista bez duplikatów: " + namesSet);
+
+        Collections.reverse(namesList);
+        System.out.println("Elementy w odwrotnej kolejności: " + namesList);
+
+        replaceElement(namesList,"Anna", "Joanna");
```

Comment:
To wyniesienie do osobnej metody jest zbędne, ponieważ nic nie upraszcza.
Szczególnie, że w linijce 15 masz analogiczny kod, a jednak nie wyniosłeś go do osobnej metody.

Reply:
"W zadaniu należy zmieni imię Anna na Joanna, ale metoda, która będzie odpowiadała za implementację tego punktu powinna być napisana, tak, żeby w łatwy sposób można było ją zastosować do innych imion." - stad wyniesienie do osobnej metody, zmienilem jednak na wersje o ktorej mowisz